src/nemos/regularizer.py

```diff
@@ -273,16 +133,32 @@ class are defined in the `allowed_solvers` attribute.
         "GradientDescent",
         "BFGS",
         "LBFGS",
-        "ScipyMinimize",
-        "NonlinearCG",
-        "ScipyBoundedMinimize",
         "LBFGSB",
+        "NonlinearCG",
+        "ProximalGradient",
     )
 
+    _default_solver = "GradientDescent"
+
     def __init__(
-        self, solver_name: str = "GradientDescent", solver_kwargs: Optional[dict] = None
+        self,
     ):
-        super().__init__(solver_name, solver_kwargs=solver_kwargs)
+        super().__init__()
+
+    def penalized_loss(self, loss: Callable):
+        """Unregularized method does not add any penalty."""
+        return loss
+
+    def get_proximal_operator(
+        self,
+    ) -> ProximalOperator:
+        """Unregularized method has no proximal operator."""
+
+        def prox_op(params, hyperparams, scaling=0.5):
+            Ws, bs = params
+            return jaxopt.prox.prox_none(Ws, hyperparams, scaling=scaling), bs
+
+        return prox_op
```

Comment:
```suggestion
```suggestion
        return jaxopt.prox.prox_none
```

This is probably sufficient since prox_none does nothing to both Ws and bs
```